docs/guides-agent-app.md

```diff
@@ -0,0 +1,433 @@
+
+Note that if we had chosen the **Ethereum Mainnet** as the network for our organization we would have passed `aragon:mainnet` instead of `aragon:rinkeby` as the argument to `--environment`.
+
+The `--apm.ipfs.rpc` option allows us to point to an IPFS node that has the files we are looking for. In our case we're pointing it to the aragon network IPFS node.
+
+We can also run our own IPFS node by running `aragon ipfs` in another terminal window. 
+
+Running a local IPFS node allows us to run the same command without the `--apm.ipfs.rpc` option (since the `--apm.ipfs.rpc` option defaults to `http://localhost:5001`). 
+
+However, since IPFS propogation is slow, it's better to point directly to the aragon IPFS node.
+
+**Note that, in the true spirit of democracy, this step will trigger a vote in the DAO, you'll need to vote *yes* to confirm the installation of the Agent app.**
+
+1. Click on the  Voting app icon on the left. You should you have one open vote.
+
+<p align="center">
+   <img width="800" src="/docs/assets/agent-guide/agent-0.png">
+</p>
+
+2. Click on the **View vote** button. You should see a panel pop up on the right hand side.
+
+<p align="center">
+   <img width="800" src="/docs/assets/agent-guide/agent-1.png">
+</p>
+
+3. Scroll to the bottom of the panel and click on the big green **Yes** button.
+
+<p align="center">
+   <img width="800" src="/docs/assets/agent-guide/agent-2.png">
+</p>
+
+4. Sign the transaction and voila! That's all there is to it. When you click on the Voting app again you should see the vote has passed with a 100% Yes vote.
+
+<p align="center">
+   <img width="800" src="/docs/assets/agent-guide/agent-3.png">
+</p>
+
+## 3. Set permissions
+
+If you look at the end of the output of the `dao install` command you ran in the previous step, you should see the following:
+```
+ℹ Successfully executed: "Execute desired action as a token holder"
+ ⚠ After the app instance is created, you will need to assign permissions to it for it appear as an app in the DAO
+```
+What does this mean exactly 
+😕? 
+
+It's telling us that although we've successfully installed the Agent app, before we can use it as part of our DAO we need to define who can access the app's functionality.
+
+In other words, we need to define who (or which app) has permission to execute actions in the Agent app and who can re-grant and revoke that permission.
+
+In this guide we're going to give the Voting app permissions to execute actions on behalf of the Agent app, and therefore on behalf of the DAO.
+
+To assign these permissions we need to get a hold of the Ethereum address of the Agent app -- remember **Agent is a fully-fledged Ethereum account** -- as well as the address of the Voting app.
+
+To do this we'll use the [`dao apps`](https://hack.aragon.org/docs/cli-dao-commands#dao-apps) command.
```

Comment:
This section could be omitted pretty soon, in the next release of the `aragonCLI` we will output the app address  after `dao install`. Maybe something to take it in consideration to reduce the load of the guide?

Reply:
Keep until next release?

Reply:
P.S. Why not assign unique human readable names to apps?
`<dao name>-<app-name>`, or something like that...

For example, If your DAO is called gabi, and you want to reference your voting app, you could write `gabi-voting` instead of the address.